zkvm/src/predicate.rs

```diff
@@ -68,62 +117,51 @@ impl Predicate {
         PointOp {
             primary: None,
             secondary: Some(h),
-            arbitrary: vec![(-Scalar::one(), self.to_point())],
+            arbitrary: Vec::new(),
         }
     }
 }
 
 impl PredicateWitness {
-    pub fn to_point(&self) -> CompressedRistretto {
-        self.to_uncompressed_point().compress()
-    }
-
-    fn to_uncompressed_point(&self) -> RistrettoPoint {
```

Comment:
we don't use it yet and VerificationKey API does not provide uncompressed point. So lets refactor this when we actually need uncompressed points.